Author: Daniel Baumann <daniel@debian.org>
Description: Disable modules check to make it buildable without X11.

diff -Naurp openerp-client.orig/setup.py openerp-client/setup.py
--- openerp-client.orig/setup.py	2009-06-08 15:50:25.000000000 +0000
+++ openerp-client/setup.py	2009-07-26 10:22:32.000000000 +0000
@@ -130,7 +130,7 @@ def translations():
         trans.append((dest % (lang, name), po))
     return trans
 
-check_modules()
+#check_modules()
 
 if os.name <> 'nt' and sys.argv[1] == 'build_po':
     os.system('(cd bin ; find . -name \*.py && find . -name \*.glade | xargs xgettext -o po/%s.pot)' % name)
