Author: Chris Lamb <lamby@debian.org>
Description: Fixes startup problem (Closes: #481895).

diff -Naurp openerp-client.orig/bin/openerp-client.py openerp-client/bin/openerp-client.py
--- openerp-client.orig/bin/openerp-client.py	2008-11-03 20:53:38.000000000 +0000
+++ openerp-client/bin/openerp-client.py	2008-11-09 16:40:03.000000000 +0000
@@ -84,6 +84,8 @@ pygtk.require('2.0')
 import gtk
 import gtk.glade
 
+gtk.gdk.threads_init()
+
 import locale, gettext
 
 import atk
